frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-node-manager/rpc_commit/readme.md

```diff
@@ -170,448 +121,266 @@ curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig
-            "node-result": [
-                {
-                    "node-id": "R1",
-                    "configuration-status": "complete"
-                },
-                {
-                    "node-id": "R2",
-                    "configuration-status": "fail",
-                    "error-message": "RemoteDevice{vnf212}: RPC during tx failed. Error messages: [/alias[name='^new']/expansion is not configured]\n",
-                    "error-type": "uniconfig-error"
-                }
-            ]
-        },
-        "overall-status": "fail"
-    }
-}
-```
-
-### Failed Example
-
-If the RPC input does not contain the target nodes and there weren't any
-touched nodes, the request will result in an error.
-
-```bash RPC Request
-curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:commit' \
---header 'Accept: application/json' \
---header 'Content-Type: application/json' \
---data-raw '{
-    "input": {
-        "target-nodes": {
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "operation-failed",
+        "error-message": "RemoteDevice{vnf212}: RPC during tx failed. Error messages: [/alias[name='^new']/expansion is not configured]",
+        "error-info": {
+          "node-id": "R1",
+          "configuration-status": "fail"
         }
-    }
-}'
-```
-
-```json RPC Response, Status: 200
-{
-    "output": {
-        "error-message": "There aren't any nodes specified in input RPC and there aren't any touched nodes.",
-        "overall-status": "fail"
-    }
+      }
+    ]
+  }
 }
 ```
 
 ### Failed Example
 
-RPC commit input has two target nodes and the flag to skip unreachable nodes. The output describes the result
-of the commit.
+RPC commit input has the flag to skip unreachable nodes.
 
 ```bash RPC Request
 curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:commit' \
 --header 'Accept: application/json' \
 --header 'Content-Type: application/json' \
 --data-raw '{
     "input": {
-        "skip-unreachable-nodes": true,
-        "target-nodes": {
-            "node": ["dev1","dev2"]
-        }
+        "skip-unreachable-nodes": true
     }
 }'
 ```
 
-```json RPC Response, Status: 200
+```json RPC Response, Status: 502
 {
-    "output": {
-        "node-results": {
-            "node-result": [
-                {
-                    "node-id": "dev1",
-                    "configuration-status": "complete"
-                },
-                {
-                    "node-id": "dev2",
-                    "error-message": "Node dev2 is unreachable (skipped)",
-                    "configuration-status": "fail"
-                }
-            ]
-        },
-        "overall-status": "fail"
-    }
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "southbound-no-connection",
+        "error-message": "Node dev2 is unreachable (skipped)",
```

Comment:
is this behaviour correct? I thought that if skip-unreachable-nodes is set to true, then commit will finish without error

Reply:
No it is not. It will be fixed in new PR.